src/App.tsx

```diff
@@ -43,6 +44,8 @@ const routes =createBrowserRouter([
       {path:"ads",element:<Ads/>},
       {path:"book",element:<Bookings/>},
       {path:"room",element:<Rooms/>},
+      {path:"add-room",element:<AddRoom/>},
```

Comment:
it would be better to make this path `rooms/add-room` for better website hierarchy  and to indicate that we reach to the `add-room` page from the rooms page.